.github/workflows/issue-label-assign.yml

```diff
@@ -53,58 +53,66 @@ jobs:
           area-is-keyword: true
           excluded-labels: "[contribution/core]"
           parameters: ${{ env.AREA_PARAMS }}
+          affixes: ${{ env.AREA_AFFIXES }}
 
 env:
  OSDS_DEVS: >
            {
              "assignees":["NGL321","peterwoodworth","ryparker"]
            }
 
+ AREA_AFFIXES: >
+           {
+             "prefixes":["@aws-cdk/"]
+           }
+    
+
  AREA_PARAMS: >
```

Comment:
It would be better to add the new keywords rather than replace the old ones. That way the automation will still function with the old keywords if people try to use them

Reply:
@NGL321 `affixes` will only add keywords to the keywords checked; it won't replace the keywords submitted by the user.

Check this test to see what is expected by design https://github.com/aws-github-ops/aws-issue-triage-manager/blob/014ef81c13ed00e8289374a8847cceb9bc706b01/src/__tests__/issue.test.ts#L523

This warrants beefing up the documentation on affixes

Reply:
I suppose that the arbitrary line selection was too vague. I follow the function of affixes, I will select some of the keywords that I think should remain